Magento_CustomerCustomAttributes/templates/customer/form/edit.phtml

```diff
@@ -31,19 +31,19 @@
         <?php if ($_taxvat->isEnabled()): ?>
             <?= /* @escapeNotVerified */ $_taxvat->setTaxvat($block->getCustomer()->getTaxvat())->toHtml() ?>
         <?php endif ?>
-        
+
         <?php if ($_gender->isEnabled()): ?>
             <?= /* @escapeNotVerified */ $_gender->setGender($block->getCustomer()->getGender())->toHtml() ?>
         <?php endif ?>
 
         <div class="checkbox dashboard-form__divider">
-            <input type="checkbox" 
-                   name="change_email" 
-                   id="change-email" 
-                   data-role="change-email" 
-                   value="1" 
-                   title="<?= /* @escapeNotVerified */ __('Change Email') ?>" 
-                   class="checkbox__field" 
+            <input type="checkbox"
+                   name="change_email"
+                   id="change-email"
+                   data-role="change-email"
+                   value="changeEmail"
```

Comment:
why we are changing the value?

Reply:
searched about `value` and here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#Value they said `This is never seen on the client-side, but on the server, this is the value given to the data submitted with the checkbox's name. ` and I've already edited the name to `account_update` and the `value` now describe what was changed. I think it's ok for now